File: 1. Iniciante/URI1041.py
a,b = input().split(" ")

x = float(a)
y = float(b)

if x > 0.00 and y > 0.00:
    print("Q1")
elif x > 0.00 and y < 0.00:
    print("Q4")
elif x < 0.00 and y > 0.00:
    print("Q2")
elif x < 0.00 and y < 0.00:
    print("Q3")
elif x == 0.00 and y ==0.00:
    print("Origem")
elif x == 0.00 and y > 0.0:
    print("Eixo Y")
elif x == 0.00 and y < 0.00:
    print("Eixo Y")
elif x > 0.00 and y == 0.00:
    print("Eixo X")
elif x < 0.00 and y == 0.00:
    print("Eixo X")

File: 1. Iniciante/URI1041.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main() {
    double x, y;

    cin >> x >> y;

    if (x > 0.00 && y > 0.00){
        printf("Q1\n");
    } else if ( x > 0.00 && y < 0.00){
        printf("Q4\n");
    } else if ( x < 0.00 && y > 0.00) {
        printf("Q2\n");
    } else if ( x < 0.00 && y < 0.00){
            printf("Q3\n");
    } else if (x == 0.00 && y ==0.00){
        printf("Origem\n");
    } else if (x == 0.00 && y > 0.00) {
        printf("Eixo Y\n");
    } else if (x == 0.00 && y < 0.00) {
        printf("Eixo Y\n");
    }else if (x > 0.00 && y == 0.00) {
        printf("Eixo X\n");
    } else if (x < 0.00 && y == 0.00) {
        printf("Eixo X\n");
    }

    return 0;
}
